contents/docs/how-posthog-works/queries.mdx

```diff
@@ -49,7 +49,7 @@ A unique persons query for pageviews now results in `1` unique user.
 
 This section covers how PostHog filters out events based on person properties.
 
-> **Note:** Currently, we use the latest person properties and join them during queries. In the future (or if you have person on events enabled) this works as follows.
+> **Note:** If you signed up before June 2024 you can configure in [settings](https://us.posthog.com/settings/project-product-analytics#persons-on-events) to use the faster queries mode instead of joining with the person table for current properties.
```

Comment:
```suggestion
> **Note:** If you signed up before June 2024, you can configure in [settings](https://us.posthog.com/settings/project-product-analytics#persons-on-events) to use the faster queries mode instead of joining with the person table for current properties. This is the default for users signed up after June 2024.
```